src/api/spec/features/webui/packages_spec.rb

```diff
@@ -374,27 +361,21 @@
       end
     end
 
-    describe 'in a project not owned by the user, eg. global namespace' do
+    describe 'in a project not owned by the user' do
       let(:admin_user) { create(:admin_user, :with_home) }
       let(:global_project) { create(:project, name: 'global_project') }
 
-      it 'as non-admin user' do
+      it 'forbidden as user' do
```

Comment:
This should be a spec for policies since it's only testing authorization. Anyway, it's already a bit better now, it's at least doing less.

Reply:
true